code/src/reducers/tasks.js

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+	list: {
+		name: 'To Do List',
+		items: [
+			// {
+			// 	id: 0,
+			// 	text: 'Eat breakfast',
+			// 	isComplete: true,
+			// },
+			// {
+			// 	id: 1,
+			// 	text: 'Go to gym',
+			// 	isComplete: false,
+			// },
+			// {
+			// 	id: 3,
+			// 	text: 'Take a nap',
+			// 	isComplete: false,
+			// },
+		],
+	},
+};
+
+const tasks = createSlice({
+	name: 'tasks',
+	initialState: initialState,
+	reducers: {
+		addTask: (state, action) => {
+			const itemInfo = action.payload;
```

Comment:
declaring a new variable for the payload could be skipped but it works all the same and gives a little bit more clarity